README.md

```diff
@@ -64,76 +46,44 @@ producing complex, multi-container applications.
         # run following command to configure your shell:
```

Comment:
output of the `vagrant service-manager env docker` needs to be updated to match the latest output. 
Also value of `DOCKER_CERT_PATH` needs to have a dummy path.